components/NavBar/index.tsx

```diff
@@ -42,18 +32,18 @@ const Navbar = () => {
             />
           </a>
         </Link>
-        <nav className='flex items-center space-x-10'>
-          {links.map(nav => (
-            <NavLink key={nav.page} {...nav} />
-          ))}
-        </nav>
-        <button
-          className='px-4 py-1 font-serif text-xl text-white hover:opacity-75'
-          onClick={toggleSignIn}
-        >
-          {/* need to add proper state when auth was added */}
-          {signIn ? 'Sign-out' : 'Sign-in'}
-        </button>
+        <div className='flex justify-end w-screen'>
+          <div className='flex w-80 justify-around  '>
+            <DropDownMenu links={links} />
+            <button
+              className='px-3 font-serif text-2xl text-white hover:opacity-75 focus:outline-none focus:ring focus:ring-teal focus:ring-opacity-50 rounded'
+              onClick={toggleSignIn}
+            >
+              {/* need to add proper state when auth was added */}
+              {signIn ? 'SIGN-OUT' : 'SIGN-IN'}
+            </button>
+          </div>
+        </div>
```

Comment:
u removed the semantic HTML and then added `w-screen` inside a padded element, which can cause issues with horizontal scrolling